ConeSearch.tex

```diff
@@ -167,11 +170,19 @@ \section{Service Interface Requirements}
 				\item[Example]
 					\url{http://mycone.org/cgi-bin/search?RA=180.567&DEC=-30.45&SR=0.0125}
 			\end{bigdescription}
-		\item As defined by DALI a service SHOULD also understand the following parameter:
+		\item As defined by DALI a service SHOULD also understand the following parameters:
 			\begin{description}
 				\item[\textbf{MAXREC}] to let the client limit the number of records returned
-					or require a service metadata response. Its usage is encouraged and preferred
-					to the SR=0 solution for metadata discovery.
+					or require a service metadata response. Its usage with a value of 0 is encouraged
+					and preferred to the SR=0 solution for metadata discovery
+					\footnote{However, keep in mind that, for back-compatibility reasons, MAXREC
+					is optional and thus SR=0 needs to be accepted for metadata probing.}.
```

Comment:
So, I guess that `MAXREC=0&SR=0` is equivalent `SR=0` ; the user will get the metadata.

But what happens if we have just `MAXREC=0`? Should we get an error (because `SR` is not 0) or just the metadata?

Reply:
@molinaro-m you are quite right; my comment missed the point that SR is mandatory.